pkg/test/resourcefixture/testdata/basic/accesscontextmanager/v1beta1/accesscontextmanagerserviceperimeter/dependencies.yaml

```diff
@@ -15,16 +15,11 @@
 apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
 kind: Project
 metadata:
-  annotations:
-    cnrm.cloud.google.com/deletion-policy: "abandon"
```

Comment:
Can we (easily?) split out the changes to the tests?  The issue is that this PR then comingles two different ideas: (1) changing/fixing the tests and (2) generating the test golden output / casettes.

The second one is easy to merge, the first one needs a lot more consideration.  So now your "fast merge" PR becomes a "slow merge" PR

Reply:
In this case, as there's only two, I think it's less work to just do it in one PR.  Can you (briefly) explain the reason for this change (and the other accesscontextmanager one), just so we have a record?